R/arguments.R

```diff
@@ -25,7 +25,7 @@ wsc_parse_args <- function(option_list, mandatory=c()){
 #' Parse numeric parameters and make sure vectors are the right length
 #'
 #' Sometimes a comma-separated string field needs to be parsed to a numeric vector.
-#'
+#' Sometimes a "a:b" expression has to be converted to a list
```

Comment:
Don't need this - just remove 'comma-separated' from the above to make it less specific.